syntax = "proto3";

package hellosvc;

service HelloSvc {
    rpc hello(HelloRequest) returns (HelloResponse);
}

message HelloRequest {
    string name = 1;
}

message HelloResponse {
    string greeting = 1;
}
